### PR TITLE
LPS-169003 Migrate Staging tree to ClayTreeView

### PR DESCRIPTION
**Pull request by sandrodw3**
Leaving this on hold as we have to send it to Staging Team after reviewing it and passing tests

/cc @ealonso

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:staging

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by Tim-Cao** [comment]
Failure is unrelated to changes in this PR. @sandrodw3

File: modules/apps/staging/staging-taglib/src/main/resources/META-INF/resources/js/PagesTree.js [hunk 1]
```diff
@@ -48,10 +48,10 @@ export default function PagesTree({
...
 			return fetch(loadMoreItemsURL, {
 				body: Liferay.Util.objectToURLSearchParams({
-					[`${namespace}parentLayoutId`]: item.layoutId,
-					[`${namespace}privateLayout`]: privateLayout,
-					[`${namespace}selPlid`]: item.plid,
-					[`${namespace}start`]: cursor * maxPageSize,
+					[`parentLayoutId`]: item.layoutId,
+					[`privateLayout`]: privateLayout,
+					[`selPlid`]: item.plid,
+					[`start`]: cursor * maxPageSize,
```
Comment (p2kmgcl) [inline modules/apps/staging/staging-taglib/src/main/resources/META-INF/resources/js/PagesTree.js:55]: If we do not need namespaces, we can replace this with:
```js
{
  parentLayoutId: item.layoutId,
  /** ... **/
}
```

Comment (sandrodw3) [reply modules/apps/staging/staging-taglib/src/main/resources/META-INF/resources/js/PagesTree.js:55]: True :heart_eyes:

**Comment by sandrodw3** [comment]
https://github.com/liferay-staging/liferay-portal/pull/201

**Pull request closed without merging**